docs/migration/5_0.md

```diff
@@ -40,3 +40,12 @@ under the section `headerDisplayOnly`.
 #### Translations (i18n) changes
 
 - `configurator.addToCart.buttonDisplayOnly` was added to `configurator-common.ts`
+
+### AsmAuthService
+
+- `coreLogout()` method was modified to trigger a global message by default. `showGlobalMsg: false` parameter can be passed to opt-out of the default global message.
+
+### AuthService
+
+- One new required constructor parameter `globalMessageService: GlobalMessageService`
```

Comment:
```suggestion
- Added `GlobalMessageService` to constructor.
```

Reply:
@psmul Is this okay:

`GlobalMessageService` is a new, required constructor dependency.